frontend/encore/reactjs.rst

```diff
@@ -23,3 +23,83 @@ That's it! Your ``.js`` and ``.jsx`` files will now be transformed through
 ``babel-preset-react``.
 
 .. _`babel-preset-react`: https://babeljs.io/docs/plugins/preset-react/
+
+Building a component
+--------------------
+
+Now that Encore can manage any React instance, let's build a simple component
+which display ``Hello World from React !``.
+
+In order to do this, let's add a new ``react.js`` file inside the ``assets`` folder:
```

Comment:
... to do this, add a new

We try not to use "let's" or "we". I allowed it on the previous paragraph... because it sounds good - but try to avoid it ;).

And how about `hello.js` instead of `react.js` - I think that filename could be confusing.

`assets` -> `assets/`